content/doc/reference/filters.dmark

```diff
@@ -484,11 +484,55 @@ title: "Filters"
   #listing[lang=ruby]
     filter :sass
 
-  #p The options that are given to this filter will be passed on to %code{Sass::Engine#initialize}. For example, this filter call set %code{:style} to %code{:compact}:
+  #p The options that are given to this filter will be passed on to %code{Sass::Engine#initialize}, to the exception of the %code{sourcemap_path} option. For example, this filter call set %code{:style} to %code{:compact}:
 
   #listing[lang=ruby]
     filter :sass, style: :compact
 
+  #p You can evaluate Ruby code from within your Sass code using the %code{nanoc()} Sass function. It takes a string containing Ruby code (a %code{Sass::Script::Value::String}):
+
+  #listing[lang=sass]
+    /* This is the input Sass code */
+    .title {
+      content: nanoc('@config[:title]');
+    %}
```

Comment:
This example is the same as on the other page.

Maybe we can give people ideas by showing they can use `nanoc()` along with `if` statements.

My commit message has an example to enable auto numbering of headings but it's too large to fit the doc site layout 😐

```
    @if nanoc('@config.dig(:items, :headings, :auto_number)') {
       body { counter-reset: h2; }
       h2 { counter-reset: h3; }
       h3 { counter-reset: h4; }
       h4 { counter-reset: h5; }
       h5 { counter-reset: h6; }

       h2:before { counter-increment: h2; content: counter(h2) ". "; }
       h3:before { counter-increment: h3; content: counter(h2) "." counter(h3) ". "; }
       h4:before { counter-increment: h4; content: counter(h2) "." counter(h3) "." counter(h4) ". "; }
       h5:before { counter-increment: h5; content: counter(h2) "." counter(h3) "." counter(h4) "." counter(h5) ". "; }
       h6:before { counter-increment: h6; content: counter(h2) "." counter(h3) "." counter(h4) "." counter(h5) "." counter(h6) ". "; }

       h2.nocount:before, h3.nocount:before, h4.nocount:before, h5.nocount:before, h6.nocount:before { content: ""; counter-increment: none; }
    }
```

So maybe something less verbose that snoops into `@config` and decides to compile out some Sass code depending on the value?